locales/templates/app.pot

```diff
@@ -0,0 +1,1205 @@
+#. name of the app (e.g. Firefox Focus)     %2$s will be replaced with a URL
+#. linking to the MPL (e.g. https://www.mozilla.org/en-US/MPL/)
+msgctxt "your_rights_content2"
+msgid ""
+"%1$s is made available to you under the terms of the <a "
+"href=\"%2$s\">Mozilla Public License</a> and other open source licenses."
+msgstr ""
+
+#. Text shown in the "Your rights" screen.     %1$s will be replaced with the
+#. name of the app (e.g. Firefox Focus)     %2$s will be replaced with a URL
+#. linking to the trademark policy (e.g.
+#. https://www.mozilla.org/foundation/trademarks/policy/)
+msgctxt "your_rights_content3"
+msgid ""
+"You are not granted any rights or licenses to the trademarks of the "
+"Mozilla Foundation or any party, including the Mozilla, Firefox or %1$s "
+"names or logos. Additional information may be found <a "
+"href=\"%2$s\">here</a>."
+msgstr ""
+
+#. Text shown in the "Your rights" screen.     %1$s will be replaced with the
+#. name of the app (e.g. Firefox Focus)     %2$s will be replaced with a URL
+#. linking to the list of licenses used by the dependencies.
+msgctxt "your_rights_content4"
+msgid ""
+"Additional source code for %1$s is available under various other free and"
+" open source <a href=\"%2$s\">licenses</a>."
+msgstr ""
+
+#. Text shown in the "Your rights" screen.     %1$s will be replaced with the
+#. name of the app (e.g. Firefox Focus)     %2$s will be replaced with a URL
+#. linking to the GPL     %3$s will be replaced with a URL linking to the
+#. tracking protection wiki page (e.g.
+#. https://wiki.mozilla.org/Security/Tracking_protection#Lists)
+msgctxt "your_rights_content5"
+msgid ""
+"%1$s also uses blocklists provided by Disconnect, Inc. as separate and "
+"independent works under the <a href=\"%2$s\">GNU General Public License "
+"v3</a>, and available <a href=\"%3$s\">here</a>."
+msgstr ""
+
+#. Focus video strings
+msgctxt "menu_drawer_home"
+msgid "Home"
+msgstr ""
+
+#. Home tile strings
+msgctxt "tile_youtube_tv"
+msgid "YouTube"
+msgstr ""
+
+msgctxt "tile_google_video_search"
+msgid "Video Search"
+msgstr ""
+
+msgctxt "tile_imdb"
+msgid "IMDB"
+msgstr ""
+
+msgctxt "tile_rottentomatoes"
+msgid "Rotten Tomatoes"
+msgstr ""
+
+msgctxt "tile_metacritic"
+msgid "metacritic"
+msgstr ""
+
+msgctxt "tile_fandango"
+msgid "Fandango"
+msgstr ""
+
+msgctxt "tile_hollywood_reporter"
+msgid "Hollywood Reporter"
+msgstr ""
+
+msgctxt "tile_flickr"
+msgid "Flickr"
+msgstr ""
+
+msgctxt "tile_instagram"
+msgid "Instagram"
+msgstr ""
+
+msgctxt "tile_pinterest"
+msgid "Pinterest"
+msgstr ""
+
+#. Settings strings
+msgctxt "settings_cookies_dialog_title"
+msgid "Clear all cookies and site data"
+msgstr ""
+
+msgctxt "settings_cookies_dialog_content"
+msgid ""
+"Selecting ‘Clear Now’ will clear all cookies and site data stored by "
+"Firefox for Fire TV. This may sign you out of websites and remove offline"
+" web content."
+msgstr ""
+
+#. Appended by save_robotranslations.py
```

Comment:
This comment doesn't make sense to me - probably better to update it (or remove)